From 10884c9e10124b6831f599956b0860f7b4ad6e4b Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Thu, 31 Aug 2023 20:20:17 +0200
Subject: [PATCH 74/83] arm64: dts: imx8mp: Describe VDD_ARM run and standby
 voltage for DH i.MX8M Plus DHCOM SoM

Describe VDD_ARM (BUCK2) run and standby voltage in DT.

Upstream-Status: Backport [e306d386cc0602d8adf6ef44e8213a0c72833c34]
Signed-off-by: Marek Vasut <marex@denx.de>
Reviewed-by: Marco Felsch <m.felsch@pengutronix.de>
Signed-off-by: Shawn Guo <shawnguo@kernel.org>
---
 arch/arm64/boot/dts/freescale/imx8mp-dhcom-som.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-dhcom-som.dtsi b/arch/arm64/boot/dts/freescale/imx8mp-dhcom-som.dtsi
index 18f342a498a03..86245099c3da7 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-dhcom-som.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mp-dhcom-som.dtsi
@@ -294,6 +294,8 @@ buck1: BUCK1 {	/* VDD_SOC (dual-phase with BUCK3) */
 			};
 
 			buck2: BUCK2 {	/* VDD_ARM */
+				nxp,dvs-run-voltage = <950000>;
+				nxp,dvs-standby-voltage = <850000>;
 				regulator-min-microvolt = <850000>;
 				regulator-max-microvolt = <1000000>;
 				regulator-ramp-delay = <3125>;
-- 
2.42.0

